### ISSUE TITLE
Live testing: InstructorCourseDetailsPageUiTest because Gmail doesn't allow access to inbox

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/repo#4633]
Seems like our code cannot access the inbox to check emails due to a recent restriction in Gmail security.
Line 192 `assertTrue(didStudentReceiveReminder(courseId, charlie.email, charliePassword));`

**Comment by wkurniawan07** [comment]
Still passing on my staging with the same "allow access for less secure apps" trick. I found a bug with this test (not related to this issue) while doing this though.

**Comment by damithc** [comment]
Document it in dev docs?

**Comment by wkurniawan07** [comment]
? It didn't fail before? From what I've gathered this "allow access for less secure app" thingy has been there since at least 2014 (which is why I said "the same" trick).
Also this fits more to the troubleshooting guide, I think.

**Comment by damithc** [comment]
OK, I realized what happened. This time I created new test accounts because the previous ones were not linked to my own HP number (for security verification). The previous test accounts (created by past devs) must have been set to allow access for less secure apps by default or by past devs. I think new Gmail accounts are stricter and default to 'not allow', so put this in the set up guide? If not, can put in troubleshooting guide too.
